tier/terraform/aws/deps.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: plural.sh/v1alpha1
+kind: Dependencies
+metadata:
+  description: tier aws setup
+  version: 0.1.0
+spec:
+  dependencies:
+  - name: aws-bootstrap
+    repo: bootstrap
+    type: terraform
+    version: '>= 0.1.1'
```

Comment:
**suggestion:** add other providers dependencies `gcp-bootstrap` and `azure-bootstrap` and make them optional. `aws` also should be optional